lib/services/messenger/translators/front.ts

```diff
@@ -589,7 +589,7 @@ export class FrontTranslator extends TranslatorScaffold implements Translator {
 			switch (details.event.type) {
 				// If this is an edit, then create a murmur to that effect
 				case 'move':
-					metadata = TranslatorScaffold.emptyMetadata(`This thread was edited by ${details.author}.`);
+					metadata = TranslatorScaffold.emptyMetadata(`This thread was ${details.event.type}d by ${details.author}.`);
```

Comment:
NIT: This is going to be whacky English coming out depending on the event type. How about `Thread alteration of type ${details.event.type} by ${details.author}.`?

Reply:
Correct, and when we encounter such things I expect them to be covered by the `switch case`, I'll comment to that effect.